src/convertToObject.js

```diff
@@ -1,12 +1,68 @@
 'use strict';
 
-/**
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString === '') {
+    return {};
+  }
+
+  const firstObrabotka = sourceString;
```

Comment:
Variable name 'firstObrabotka' is not descriptive of its purpose. Consider renaming it to reflect that it holds the source string for processing.